src/app/not-found.tsx

```diff
@@ -1,5 +1,45 @@
+import Image from "next/image";
+import Button from "@/components/Button";
+import Logo from "@/public/Footer.webp";
+import Link from "next/link";
+import { links } from "@/data/Footer";
+
 const NotFound = () => {
-  return <div>404 Page Not Found</div>;
+  return (
+    <div className="flex h-[820px] w-full flex-col items-center justify-center bg-gradient-to-b from-hlg-dark-gray via-hlg-dark-gray to-hlg-red-200">
+      <Image
+        src={Logo}
+        alt="Highlander Gloves Logo"
+        className="absolute left-8 size-[620px] translate-y-20 transform items-end"
```

Comment:
is it possible to use % values for `size`?

Reply:
when I try using %, it makes the photo compressed

Reply:
ok np